lib/velum/dex/oidc.rb

```diff
@@ -0,0 +1,24 @@
+require "base64"
+
+module Velum
+  # This class offers the integration between ruby and the Saltstack API.
+  module Dex
+    class << self
+      def oidc_connectors_as_pillar
+        DexConnectorOidc.all.map do |con|
+          {
+            "type"          => "oidc",
```

Comment:
It's generally recommended to use the new json format.

Reply:
Changed in 4f7811b